tests/RetryTest.php

```diff
@@ -109,14 +109,14 @@ private function waitForDeadConnection(): void
     private function getConnection(): PDO
     {
         $this->dbParams = [
-            'user' => getenv('TEST_RDS_USERNAME'),
-            '#password' => getenv('TEST_RDS_PASSWORD'),
-            'host' => getenv('TEST_RDS_HOST'),
-            'database' => getenv('TEST_RDS_DATABASE'),
-            'port' => getenv('TEST_RDS_PORT'),
+            'user' => getenv('TEST_RDS_USERNAME') === false ? null : getenv('TEST_RDS_USERNAME'),
```

Comment:
ma to nejakej vyznam?

Reply:
jj, když neežistoval env tak byla hodnato false a to se podle phpstanu nelíbilo PDOčku

Reply:
aha, zajimavy, ze to dostopoval az sem, no tak to muzem rovnou castnout na string, kdyz to bude prazdny tak to stejne vyfailuje pri pripojeni

Reply:
jj to by šlo - je to tam ;)